### DISCUSSION TITLE
10 - Blocks Support

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#685] [RFCs/Ideas]
This thread is to discuss Dendron's upcoming [Block Reference](https://wiki.dendron.so/notes/95f7193b-9940-42ba-841f-3e2a4d937ba3.html) implementation.

**Comment by ScriptAutomate** [comment]
I wanted to follow-up with this discussion, using the following links:

- [New RFC Location for RFC 10 - Blocks](https://docs.dendron.so/notes/95f7193b-9940-42ba-841f-3e2a4d937ba3/)
- Has been implemented! Can read the docs on:
  - [Block References](https://wiki.dendron.so/notes/f1af56bb-db27-47ae-8406-61a98de6c78c/#block-references)
  - [Block Anchors](https://wiki.dendron.so/notes/f1af56bb-db27-47ae-8406-61a98de6c78c/#block-anchor)
  - [Block Links](https://wiki.dendron.so/notes/f1af56bb-db27-47ae-8406-61a98de6c78c/#block-links)
